core/interaction_type.cpp

```diff
@@ -24,11 +24,14 @@
 #include "slim_eidos_block.h"
 #include "subpopulation.h"
 #include "slim_sim.h"
-
+#include <omp.h>
 #include <utility>
 #include <algorithm>
 
+#include <ctime>
+#include <chrono>
 
+using namespace std::chrono;
```

Comment:
I try to avoid "using" entirely, it is just asking for weird problems.  Just put "std::chrono::" where it's needed.